raportit/puutteet.php

```diff
@@ -115,8 +115,8 @@
 
   $pvm = date("Ymd");
 
-  $worksheet->writeString($excelrivi, 0, t("Puutelistaus"));
-  $worksheet->writeString($excelrivi, 1, $pvm);
+  $worksheet->writeString($excelrivi, 0, t("Puutelistaus")." $asiakasrow[nimi]"." $asiakasrow[nimitark]".", $asiakasrow[asiakasnro]");
+  $worksheet->writeString($excelrivi, 3, $pvm);
```

Comment:
Jätin tarkoituksella tyhjiä, koska kun avaa sen excelin ilman noita tyhjiä, näyttää kuin siinä ensimm. solussa lukisi vain Puutelistaus. Kun jätin noita tyhjiä, näkee heti, että siinä lukee myös asiakkaan nimi. Käyttäjäystävällisempi näin